Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM alpine:3.6
+
+WORKDIR /opt/issue-sync
+
+ADD bin/issue-sync /opt/issue-sync/issue-sync
```

Comment:
Use COPY rather than ADD. COPY is the recommended command and avoids some extra magic that ADD performs.